apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -0,0 +1,119 @@
+import FontAwesomeV6Icon from '@code-dot-org/component-library/fontAwesomeV6Icon';
+import classNames from 'classnames';
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+
+import useLifecycleNotifier from '@cdo/apps/lab2/hooks/useLifecycleNotifier';
+import {LifecycleEvent} from '@cdo/apps/lab2/utils/LifecycleNotifier';
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {DEFAULT_PACK} from '../constants';
+import {PlaybackEvent} from '../player/interfaces/PlaybackEvent';
+import MusicLibrary from '../player/MusicLibrary';
+import MusicPlayer from '../player/MusicPlayer';
+import {setIsPlaying} from '../redux/musicRedux';
+
+import moduleStyles from './ExemplarPlayer.module.scss';
+interface ExemplarPlayerViewProps {
+  playbackEvents: PlaybackEvent[];
+  title: string;
+}
+
+const ExemplarPlayerView: React.FunctionComponent<ExemplarPlayerViewProps> = ({
+  playbackEvents,
+  title,
+}) => {
+  const dispatch = useAppDispatch();
+  const isPlaying = useAppSelector(state => state.music.isPlaying);
+
+  const playerRef = useRef<MusicPlayer | null>(null);
+  if (playerRef.current === null) {
+    playerRef.current = new MusicPlayer();
```

Comment:
Just confirming that this still works correctly for artist sounds (non-120 BPM/C tracks)? Since we're no longer setting the BPM/key on this player instance, is it being updated somewhere else?

Reply:
Yes, I've tested that this works for a variety of artist sounds with different tempos/keys. The only way that the exemplar can have a different bpm/key than the student project is if the former is using the CDO sounds. In this case, both will sync to the student's project. This ensures that the audio playback matches if the student re-creates the exemplar. 
FWIW, I don't believe we'll ever actually hit this path (where `playerRef.current === null`).